tutorial.md

```diff
@@ -18,32 +18,33 @@ Open the directory where the repository is cloned as a workspace in the editor,
 ---
 **Legacy Cloud Shell Editor**
 
-* Go to the `File` menu.
-* Select `Open Workspace`.
-* Choose the directory where the repository has been cloned. This directory is the current directory in the cloud shell terminal.
+1. Go to the `File` menu.
+2. Select `Open Workspace`.
+3. Choose the directory where the repository has been cloned. This directory is the current directory in the cloud shell terminal.
 
 **New Cloud Shell Editor**
 
-* Go the hamburger icon located in the top left corner of the editor.
-* Go to the `File` Menu.
-* Select `Open Folder`.
-* Choose the directory where the repository has been cloned. This directory is the current directory in the cloud shell terminal.
+1. Go the hamburger icon located in the top left corner of the editor.
+2. Go to the `File` Menu.
+3. Select `Open Folder`.
+4. Choose the directory where the repository has been cloned. This directory is the current directory in the cloud shell terminal.
 
-## Details of your chosen three-tier web app Jump Start Solution
+## Before you begin
+
+Before editing the solution, you should be aware of the following information:
 
-* [Solution guide](https://cloud.google.com/architecture/application-development/three-tier-web-app)
 * Application code for the frontend service is available under `src/frontend`.
 * Application code for the middleware service is available under `src/middleware`.
 * Terraform / infrastructure code is available in the `*.tf` files.
 
 Both the frontend service and the middleware service are built as container images and deployed using Cloud Run.
-
+We also strongly recommend that you familiarize yourself with the three-tier web app solution by reading the [solution guide](https://cloud.google.com/architecture/application-development/three-tier-web-app).
 
 ## Edit the solution
 
-Edit the `createHandler` function in <walkthrough-editor-select-line filePath="./src/middleware/main.go" startLine="170" endLine="171" startCharacterOffset="0" endCharacterOffset="0">./src/middleware/main.go</walkthrough-editor-select-line> to add a prefix string to every TODO item by replacing `t.Title = r.FormValue("title")` with `t.Title = "Prefix " + r.FormValue("title")`.
+For example, edit the createHandler function in <walkthrough-editor-select-line filePath="./src/middleware/main.go" startLine="170" endLine="171" startCharacterOffset="0" endCharacterOffset="0">./src/middleware/main.go</walkthrough-editor-select-line> to add a prefix string to every TODO. To add the prefix, replace `t.Title = r.FormValue("title") with t.Title = "Prefix " + r.FormValue("title")`.
```

Comment:
why are we removing backticks from createHandler?

Reply:
Please check the file diff again. We are not removing backtick.

Reply:
<img width="1265" alt="Screenshot 2023-12-13 at 14 10 26" src="https://github.com/GoogleCloudPlatform/solutionbuilder-terraform-google-three-tier-web-app/assets/124978972/519da0d4-b90a-4e2e-b9ff-61c586c9f130">
the backtick has been removed from the word createHandler word in the first line